data_classes_test.py

```diff
@@ -0,0 +1,79 @@
+"""
+Unit tests for data classes
+
+Implemented:
+    Data
+
+TODO:
+    DataController
+    Dataset
+    Portfolio
+"""
+
+import unittest
+from datetime import datetime
+from thousandaire.data_classes import Data
+
+class TestData(unittest.TestCase):
+    """
+    Unit test object for Data.
+    """
+    def setUp(self):
+        self.data = Data('test', ['buy', 'sell'])
+        self.data_list = [
+            [(datetime(2020, 1, 27), 27, 27), (datetime(2020, 1, 28), 28, 28)],
+            [(datetime(2020, 1, 29), 29, 29), (datetime(2020, 1, 30), 30, 30)]]
+
+    def assert_eqaul_content(self, response, answer):
```

Comment:
Please let it be "assertEqualContent", which is consistent with "assertEqual", etc.
This will make users of this function able to write clear readable code.

Reply:
But in that format would not pass the pylint.

Reply:
I see. What a bad design.